docs/markdown/Machine-files.md

```diff
@@ -40,8 +75,8 @@ llvm-config = '/usr/lib/llvm8/bin/llvm-config'
 Cross example:
 
 ```ini
-c = '/usr/bin/i586-mingw32msvc-gcc'
-cpp = '/usr/bin/i586-mingw32msvc-g++'
+c = ['ccache', '/usr/bin/i586-mingw32msvc-gcc']
+cpp = ['ccache', '/usr/bin/i586-mingw32msvc-g++']
```

Comment:
Not sure we want to change this example, a simple string works and IIRC there is an open issue somewhere to prepend ccache automatically when available.

Reply:
The point was that a list is also accepted, as it came up recently that you can pass these as lists not just as strings, and using ccache was a reasonable example. If you've got a different example using a list I'll change it.